virttest/utils_libvirt/libvirt_vmxml.py

```diff
@@ -0,0 +1,23 @@
+"""
+Module simplifying manipulation of the vm attributes described at
+http://libvirt.org/formatdomain.html
```

Comment:
./virttest/libvirt_xml/vm_xml.py
most of operations related to vm_xml are put under above file, shall we consider add this method into this file ,rather that create one new file libvirt_vm_xml.py?

Reply:
This is recommended from dzheng --
dzhengfy 20 days ago  Contributor
This is not specific libvirt feature related. So I recommend we put it into a new file, like virttest/utils_libvirt/libvirt_vmxml.py.
BTW, libvirt.py is too big and messed up, so we decide not to add new utility functions in it any more.